components/src/md3/ProgressBar/ProgressBar.test.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { ProgressBar } from './ProgressBar';
+import { cleanup } from '@testing-library/react-native';
+import TestRenderer from 'react-test-renderer';
+
+describe('ProgressBar', () => {
+    afterEach(cleanup);
+    it('ProgressBar Renders', () => {
```

Comment:
Small thing — usually we phrase the test statement so that it reads like a sentence, so that the code reads like "it renders without crashing" instead of "it progressbar renders"

```suggestion
    it('renders without crashing', () => {
```